lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -350,10 +349,33 @@ class CategoryRenderer {
       percentageEl.title = Util.i18n.strings.errorLabel;
     }
 
+    if (this.hasApplicableAudits(category)) {
```

Comment:
I feel this could be inlined and just `category.auditRefs.some(ref => ref.result.scoreDisplayMode !== 'notApplicable')` (or at least, the body of hasApplicableAudits could be that).

Reply:
that's much better, thanks